<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginWidgets</class>
 <widget class="QWidget" name="LoginWidgets">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>427</width>
    <height>330</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>427</width>
    <height>330</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>427</width>
    <height>330</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>LoginWidgets</string>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <widget class="QPushButton" name="btn_regist">
   <property name="geometry">
    <rect>
     <x>321</x>
     <y>199</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(11, 163, 219);</string>
   </property>
   <property name="text">
    <string>注册帐号</string>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_passwd">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>227</y>
     <width>193</width>
     <height>26</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>170</width>
     <height>22</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>9</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget{

border : none;

color: rgb(0,27,47);

font: 9pt &quot;微软雅黑&quot;;

}

QWidget:hover{

border-width : 1px;

border-color : #1ea4f2;

border-style : solid;

border-width : 1px;

}</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_edit_pwd">
   <property name="geometry">
    <rect>
     <x>321</x>
     <y>229</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(11, 163, 219);</string>
   </property>
   <property name="text">
    <string>找回密码</string>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QComboBox" name="cBox_account">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>196</y>
     <width>193</width>
     <height>29</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>9</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QComboBox {

border-width : 1px;

border-color : rgb(255,255,255);

border-style : solid;

color: rgb(0,27,47);

font: 9pt &quot;微软雅黑&quot;;

}

QComboBox:hover {

border-width : 1px;

border-color : #1ea4f2;

border-style : solid;

}

QComboBox::drop-down {

/*image: url(:/CIO/Image/Logon/btn_combobox_normal.png);/*

}</string>
   </property>
   <property name="editable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_top_img">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>427</width>
     <height>185</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_user_img">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>195</y>
     <width>80</width>
     <height>80</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_login">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>290</y>
     <width>191</width>
     <height>26</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
border-radius:8px;
background-color: rgb(8,164,218);
font: 9pt &quot;微软雅黑&quot;;
wid
/*border-image: url(:/CIO/Image/Logon/button_login_normal.png);*/

}

QPushButton:hover{
background-color:rgb(230,206,149)
/*border-image: url(:/CIO/Image/Logon/button_login_hover.png);
*/
}

</string>
   </property>
   <property name="text">
    <string>登录</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>131</x>
     <y>260</y>
     <width>69</width>
     <height>15</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QCheckBox" name="checkBox_rPasswd">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>记住密码</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>258</x>
     <y>260</y>
     <width>69</width>
     <height>15</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QCheckBox" name="checkBox_autoLogin">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>自动登录</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
